### DISCUSSION TITLE
Ignore battery full-at option

### DISCUSSION THREAD

**Discussion by usr1488** [polybar/polybar#2860] [Ideas]
I found out that `full-at` option in battery module takes higher precedence that any other. So bcs of that when battery is fully charged, `label-full` will be printed despite that battery is still charging, and for example color of underline will be set as if battery is not charging anymore which is incorrect. In my case underline colors are different when battery is discharging (normal mode) and discharging, but `full-at` option breaks this behavior. Simplest way to fix this just to remove upper bound check or set to `INT_MAX` here [battery-module](https://github.com/polybar/polybar/blob/master/src/modules/battery.cpp#L29) and set `full-at` option in config for example to 101, so that we will never reach this level.

**Comment by patrick96** [comment]
Just so that I understand correctly, you want a way to not have `format-full` at all?

**Reply by usr1488** [reply]
Probably no, I just want that charging state would have higher precedence over full state. So that even if battery is full, `format-charging` will keep displaying but not `format-full`.

**Reply by patrick96** [reply]
So under what circumstance would you expect `format-full` to display? It seems the battery is always either charging or discharging?
